README.md

```diff
@@ -1,15 +1,97 @@
+The first registration will require the approval from the DODAS admins.
 
 ## Setting up dodas client
 
-## Available containers orchestrators
+Get the latest client following instructions at https://dodas-ts.github.io/dodas-go-client/.
+
+
+
+## Getting the token
+
+Follow instructions at https://indigo-iam.github.io/docs/v/current/user-guide/getting-a-token.html.
+
+Now you are ready to deploy your application.
+
+## Available applications
+All of these templates uses the helm charts defined in https://github.com/DODAS-TS/helm_charts/tree/master/stable.
+
+### K8s as a service
+
+### Spark
+Apache Spark is a fast and general-purpose cluster computing system.
+
+* http://spark.apache.org/
+
+This chart will do the following:
+
+* 1 x Spark Master with port 30808 exposed with a nodePort service (webUi)
+* 1 x Jupyter notebook with port 30888 exposed with a nodePort service, with 2 executors
+* All using Kubernetes Deployments
+
+With these templates you can deploy Apache Spark on top of either k3s or k8s:
+- [Spark on k3s](templates/applications/k3s/template-spark.yml)
+- [Spark on k8s](templates/applications/k8s/template-spark.yml)
+
+
+### HTCondor
+HTCondor is an open-source high-throughput computing software framework for coarse-grained distributed parallelization of computationally intensive tasks (https://research.cs.wisc.edu/htcondor/).
+
+With these templates you can deploy HTCondor on top of either k3s or k8s:
+- [HTCondor on k3s](templates/applications/k3s/template-htcondor.yml)
+- [HTCondor on k8s](templates/applications/k8s/template-htcondor.yml)
+
+### CachingOnDemand
+
+XCache description is available in this article [here](https://iopscience.iop.org/article/10.1088/1742-6596/513/4/042044/pdf).
+
+You can look at the [official XrootD documentation](http://xrootd.org/docs.html) for detailed information about the XRootD tool:
+
+- [basic configuration](http://xrootd.org/doc/dev47/xrd_config.htm)
+- [cmsd configuration](http://xrootd.org/doc/dev45/cms_config.htm)
+- [proxy file cache](http://xrootd.org/doc/dev47/pss_config.htm)
+
+With these templates you can deploy Caching On Demand on top of either k3s or k8s:
+- [CachingOnDemand on k3s](templates/applications/k3s/template-cachingondemand.yml)
+- [CachingOn Demand on k8s](templates/applications/k8s/template-cachingondemand.yml)
+
+
+
+
 
 ## Quick start
+Let's take Apache Spark deployment on K8s as an example. The template to be used is [this](/templates/applications/k8s/template-spark.yml). 
+To start your deployment:
+
+```
+dodas create /templates/applications/k8s/template-spark.yaml
+```
+
+The output should be like this:
+![Alt text](/images/1.png)
+
+To get the infrastructure ID (infID) of all your deployments
+```
+dodas list infIDs
+```
+And the output should be like this:
+![Alt text](/images/2.png)
 
-## Contribution guide
+To check the status of the deployment
+```
+dodas get status <infID>
+```
```

Comment:
at the end of the deployment show the screenshot of how the jupyter and kubernetes dash can be accessed

also dodas get output is useful here

Reply:
Couldn't open jupyter on bari infrastructure